README.md

```diff
@@ -58,6 +58,9 @@ itkdev_openid_connect:
         # Optional: Specify leeway (seconds) to account for clock skew between provider and hosting
         #           Defaults to 10
         leeway: '%env(int:ADMIN_OIDC_LEEWAY)%'
+        # Optional: Allow http OIDC discovery document (used for mocking a IdP)
```

Comment:
It's not just for the discovery document, but for all (server side) requests to the IdP.

Reply:
Good point. Rephrased.